lhotse/cut.py

```diff
@@ -1478,6 +1480,30 @@ def compute_and_store_features(
         ))
         return cut_set
 
+    def compute_global_feature_stats(self, storage_path: Optional[Pathlike] = None) -> Dict[str, np.ndarray]:
+        """
+        Compute the global means and standard deviations for each feature bin in the manifest.
```

Comment:
It might make sense to add a max_cuts argument, e.g. max_cuts = 5000, and have it randomly sample that many cuts if there are more than that available.  Otherwise could be wastefully slow.